x/evm/keeper/evm_state.go

```diff
@@ -35,7 +34,7 @@ type EvmState struct {
 	//    a piece of state in that contract forms the primary key.
 	//  - value (V): State value bytes.
 	AccState collections.Map[
-		AccStatePrimaryKey, // account (EthAddr) + state key (EthHash)
+		collections.Pair[gethcommon.Address, gethcommon.Hash], // account (EthAddr) + state key (EthHash)
```

Comment:
I used the raw `collections.Pair` type because it makes it more apparent that it's from the collections struct, rather than something else.